### DISCUSSION TITLE
Show insights into what shortcuts are available in the current context

### DISCUSSION THREAD

**Discussion by mathphye** [godotengine/godot-proposals#7980] [Editor]
### Describe the project you are working on

None. This is an observation of my little experience working with godot. But also I see many others facing this:

Just today, without searching it intentionally find those two:

https://github.com/godotengine/godot/issues/82501
"No obvious way to delete AnimationNodeBlendTree node in AnimationTree editor"

https://github.com/godotengine/godot-proposals/discussions/7482
"Drag and drop onto "Add frames from sprite sheet"

https://github.com/godotengine/godot/issues/80515
"When in script editor, Cmd+, is acknowledged by the menu bar but doesn't open editor settings"

### Describe the problem or limitation you are having in your project

Shortcuts are not intuitive or follow a standard (international) approach such as it won't be necessary to memorize shortcuts for each software, people usually learn it randomly from tutorials passed by peer-person recommendations. This is not efficient to advertise shortcuts.

There is a guide: https://docs.godotengine.org/en/stable/tutorials/editor/default_key_mapping.html

However people won't memorize all of it beforehand, and usually search for the shortcut that is needed in real-time.,

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Show in a bar which shortcuts are available, maybe add also the name of the current view-context.

Blender example:
![image](https://github.com/godotengine/godot-proposals/assets/146298448/b276d195-7191-4a25-aafe-b3bd699ddd4b)

This proposal wants to show real-time shortcuts available.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Show in a bar which shortcuts are available.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Not sure. Someone familiar with the code should figure out.

### Is there a reason why this should be core and not an add-on in the asset library?

Many people use shortcuts and the advertisement of them is not in real-time as needed. Would be better for all people using shortcuts to see in real-time which shortcuts are available.

**Comment by snougo** [comment]
Nodes in AnimationTree Editor without Delete Button kill me, we need improve this part of Editor UI
